examples/features/blocks.effekt.md

```diff
@@ -0,0 +1,28 @@
+# Blocks
+
+```effekt:hidden
+interface Yield[A] {
+  def yield(x: A): Unit
+}
+```
+
+As we have established in a previous section, functions in Effekt are second-class. While you cannot pass them as first-class values to other functions as argument, you can pass them as second-class blocks.
```

Comment:
I might suggest moving this whole section into the `functions.effekt.md` section... it's not entirely clear at the moment how UFCS and blocks interact (i.e. I was surprised that you can't do `{ 5.div(0) }.default(0)`)

Reply:
Good point, I will add some explanation for blocks and UFCS